pontoon/base/static/css/style.css

```diff
@@ -821,11 +822,20 @@ header .right .select .menu {
 
 .notifications .menu li.notification-item .message.trim,
 .notifications .menu li.notification-item .message.trim > p {
+    pointer-events: none;
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
 }
 
+.notifications .menu li.notification-item .message.trim a {
+    color: #aaaaaa;
+}
+
+.notifications .menu li.notification-item.hover .message.trim a {
+    color: #ebebeb;
+}
```

Comment:
We should have the same rule in the Translate view CSS file.

Reply:
The description on hover turns from #aaaaaa to #ebebeb on the normal notification bar; hence this rule needs to be added. 

Change of color does not happen on hovering in general for Translate view CSS so the rule is not needed.

Reply:
Behavior is inconsistent, though. If you have a comment notification in the Translate view, comment text doesn't become lighter, whereas in the /notifications page and in the notifications menu in a non-translate view, it does.

Styling should be consistent.